crypto/src/group_store.rs

```diff
@@ -219,7 +219,7 @@ pub(crate) const ITEM_LIMIT: u32 = 100;
 
 impl HybridMemoryLimiter {
     pub(crate) fn new(count: Option<u32>, memory: Option<usize>) -> Self {
-        let maybe_memory_limit = memory.or_else(|| {
+        let maybe_memory_limit = memory.or({
```

Comment:
`or_else` was intentional as well, to prevent eager fetching of system metrics if not needed